### PR TITLE
add missing includes

### PR DESCRIPTION
**Pull request by nim65s**
Hi,

Here are some trivial fixes which seems required to build with clang 18 from nixpkgs.

### CODE REVIEW

File: libs/utils/include/utils/memalign.h [hunk 1]
```diff
@@ -17,6 +17,7 @@
 #ifndef TNT_UTILS_MEMALIGN_H
 #define TNT_UTILS_MEMALIGN_H
 
+#include <tuple>
```
Comment (pixelflinger) [inline libs/utils/include/utils/memalign.h:20]: why is this one needed?

Comment (nim65s) [reply libs/utils/include/utils/memalign.h:20]: For the `std::ignore`, which itself avoid a warn_unused_result

Comment (pixelflinger) [reply libs/utils/include/utils/memalign.h:20]: okay let's just do `(void)` I don't want to pull a whole header just for that.

Comment (nim65s) [reply libs/utils/include/utils/memalign.h]: Ok, done

**Review by pixelflinger — APPROVED** [review]

**Pull request merged**